Dockerfile

```diff
@@ -15,14 +15,15 @@ ADD $cacert /etc/pki/ca-trust/source/anchors
 ADD https://gist.githubusercontent.com/mijaros/c9c9ed016ce9985d96c6c5c3b35b4050/raw/66587720883554b03a4c24875fa47442db231a51/ca.pem /etc/pki/ca-trust/source/anchors
 RUN update-ca-trust
 
+RUN useradd --no-log-init -u 1001 -G root -r -m default
```

Comment:
this should be `-g root` and I would really like to see also `-r` removed (the latter is mild, however unless there is good reasoning I'd prefer to make sure `$HOME` exists; actually there is reference to `$HOME later in Dockerfile, likely to eradicate pipenv cache, so it is highly desirable to ensure it is properly set with correct access permissions)